README.md

```diff
@@ -2,11 +2,15 @@
 
 <!-- Include start LXD intro -->
 # LXD
-LXD is a next generation system container and virtual machine manager.
-It offers a unified user experience around full Linux systems running inside containers or virtual machines.
+LXD is a modern, secure and powerful system container and virtual machine manager.
 
-It's image based with pre-made images available for a [wide number of Linux distributions](https://images.linuxcontainers.org)
-and is built around a very powerful, yet pretty simple, REST API.
+It provides a unified experience for running and managing full Linux systems inside containers or virtual machines. LXD supplies images for a [wide number of Linux distributions](https://images.linuxcontainers.org) and is built around a very powerful, yet pretty simple, REST API. LXD scales from one instance on a single machine to a cluster on a full data center rack, making it suitable for running workloads both for development and in production.
```

Comment:
"in a full data center rack," reads better to me. Otherwise I get a vision of an empty server rack with a bunch of LXD servers balancing precariously on the top of it :)